test/src/test/java/org/corfudb/integration/ClusterReconfigIT.java

```diff
@@ -332,6 +331,39 @@ public void restartTest() throws Exception {
         assertThat(shutdownCorfuServer(corfuServer)).isTrue();
     }
 
+    /**
+     * Test reboot utility including reset and restart server
+     * @throws Exception
+     */
+    @Test
+    public void rebootUtilTest() throws Exception {
+        final int PORT_0 = 9000;
+        final String SERVER_0 = "localhost:" + PORT_0;
+        final int retries = 3;
+
+        // Start testing restart utility
+        Process corfuServer = runPersistentServer(corfuSingleNodeHost, PORT_0, true);
+
+        runtime = createDefaultRuntime();
+        Layout layout = incrementClusterEpoch(runtime);
+        RebootUtil.restart(SERVER_0, runtime.getParameters(), retries, PARAMETERS.TIMEOUT_LONG);
+
+        waitForEpochChange(epoch -> epoch >= layout.getEpoch() + 1, runtime);
+        assertThat(shutdownCorfuServer(corfuServer)).isTrue();
+        runtime.shutdown();
+
+        // Start testing reset utility
+        corfuServer = runPersistentServer(corfuSingleNodeHost, PORT_0, true);
```

Comment:
Just a test optimization: You can reuse the same server from restart and check if the epoch returns to 0.

Reply:
Done. I was not able to reuse it because after restart I assert `assertThat(shutdownCorfuServer(corfuServer)).isTrue();`. But I think it's not necessary, so I removed it and now it can be reused.